tools/topology/v2/pipe-volume-playback.conf

```diff
@@ -0,0 +1,162 @@
+# "DefineTemplate" doesn't seem to be needed
```

Comment:
@lyakh I think we should have DefineTemplate{}. Ideally we should have DefineTemplate{} follow ll Define*{} for that kind. For example, DefineMixerControl{} should have a DefineTemplate.mixercontrol{}, with default values so that we can instantiate a mixercontrol with that template and modfied attributes (if needed) easily.

Reply:
@ranj063 https://github.com/thesofproject/sof/pull/3211 is the one to look at.